inspirehep/utils/helpers.py

```diff
@@ -115,3 +115,22 @@ def force_list(data):
     elif isinstance(data, (tuple, set)):
         return list(data)
     return data
+
+
+def clean_empty(data):
+    """
+    Implementation from http://stackoverflow.com/a/27974027/890185
+
+    Args:
+        data: a dict.
+    Returns:
+        dict: a dict with empty elements removed.
+    """
+    if not isinstance(data, (dict, list)):
+        return data
+    if isinstance(data, list):
+        return [v for v in (clean_empty(v) for v in data) if v or v == 0]
+    return {
+        k: v for k, v in ((k, clean_empty(v)) for k, v in data.items())
+        if v or v == 0
+    }
```

Comment:
Yes, they seem to do the same (one in place and the other one not). I can just reuse the existing one.